.gitignore

```diff
@@ -16,6 +16,20 @@ dap4_test/findtestserver4.c
 dap4_test/pingurl4.c
 ncdap_test/findtestserver.c
 
+Makefile.in
+aclocal.m4
+compile
+config.guess
+config.h.in
+config.sub
+configure
+depcomp
+install-sh
+ltmain.sh
+m4
+missing
+test-driver
```

Comment:
Are you sure that test--driver and ltmain.sh should be ignored?

Reply:
For the developers' git repository? Almost certainly.  For release tarballs (made with `make dist` or `make distcheck`), no, they should absolutely be included.

On a related note, what is the policy on building GitHub's automatic repository tarballs?  Do you assume everyone has a full Autotools or CMake install (the assumption I made here) or only POSIX Make and Bourne Shell (the assumption made to Autotools release tarballs)?

Reply:
We turn on maintainer mode in the release branches, so full installs should not be required.  A lot of the generated files are explicitly kept out of version tracking, and only added to the 'v[xyz]-release' branch which gets tagged.

Reply:
This will help keep the generated files out of version tracking, but will make adding them to the release branches a bit trickier unless the release branches drop this part of the `.gitignore` file; should I add a comment at the start to that effect?

E.g.
```gitignore
# Autotools-generated files
# Delete this section on release branches
```